core/domain/classifier_services_test.py

```diff
@@ -205,7 +205,11 @@ def _mock_get_label_name(l):
             self.string_classifier._get_prediction_report_for_doc(-1))
         self.assertEquals(prediction_report['prediction_label_id'], 1)
 
-    def test_training(self):
+    def test_predict_label_for_doc(self):
+        # Until I come up with larger test data sets, this will do
```

Comment:
After thinking about this for a bit, I believe it makes sense to use smaller datasets for this unit test. There's a larger dataset I'd like to use involving responses submitted to existing explorations, but I think that's beyond the scope of this PR. Updated the doc that makes a tradeoff between detail vs length, let me know if it doesn't make sense.